README.md

```diff
@@ -2,19 +2,28 @@
 
 ⚠️ **Warning: this project is still very alpha**
 
-This project is:
+React component variations are:
 * A format called "variations" for specifying React component **_examples_** and providing **_documentation_** for them.
 * Tools for finding, validating, and iterating over variations.
+* A collection of "consumers" for doing useful things with variations.
 
-A variation looks like:
+## Table of contents
+
+- [Variations](#variations)
+- [Consumers](#consumers)
+- [Contribution Guidelines](#contribution-guidelines)
+
+## Variations
+
+Variations are defined in files we call variation providers. These are standard Javascript modules that return an object with specific fields. For example:
 
 ```jsx
 // ButtonVariationProvider.jsx
 
 import React from 'react';
 import Button from '../components/Button';
 
-export default function ButtonVariationProvider({ action }) {
+export default function ButtonVariationProvider() {
```

Comment:
it seems important to have the example include the "extras" functionality, since it's so critical for storybook.

Reply:
But isn't that just one consumer of variations? It still feels like an advanced feature to me. If we want to include it in the "here's what you need to know right out of the gate" part of the readme here, I think we'd need to make sure we provide the "why" and "how" on top of the "what" as it relates to extras functionality.